model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php

```diff
@@ -101,20 +104,7 @@ private function resolve(ItemMediaResolver $resolver, string $assetUrl, string $
 
     private function getReplacementName(string $link, array $replacementList): string
     {
-        $basename = basename($link);
-
-        $replacement = $basename;
-
-        $count = 0;
-        while (in_array($replacement, $replacementList)) {
-            $dot = strrpos($basename, '.');
-            $replacement = $dot !== false
-                ? substr($basename, 0, $dot) . '_' . $count . substr($basename, $dot)
-                : $basename . $count;
-            $count++;
-        }
-
-        return $replacement;
+        return hash('crc32', basename($link));
```

Comment:
may i ask you why you want to use `crc32` and not another algorithm

Reply:
Wont this break some `decode` made in other place?